### PR TITLE
Package skeleton

### PR DESCRIPTION
**Pull request by PietrH**

### CODE REVIEW

**Comment by PietrH** [comment]
Do we want to add tests? Github actions for CMD R CHECK?

**Comment by adokter** [comment]
Thanks @PietrH, enabling an automated R CMD CHECK would be great

**Review by adokter — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,25 @@
...
+
+Skeleton package to continue sun/solar functions and methods from maptools.
+
+## Installation
+
+You can install the development version of sunMaptools from [GitHub](https://github.com/) with:
+
+``` r
+# install.packages("devtools")
+devtools::install_github("rsbivand/sunMaptools")
```
[the remark this answers is not in the document]
Comment (PietrH) [reply README.md:15]: Agreed, this was done using `usethis::use_readme_md()` , I'm not sure why it didn't pick up on adokter/suntools instead.

**Comment by PietrH** [comment]
Do we want to change the license to GPL3?

**Comment by PietrH** [comment]
Some more things to consider:

- Switching to LGPL v3
- Author list
- Version numbering

**Comment by PietrH** [comment]
I've added a placeholder test in e2e83fb64769ebf19042ef9d4c23e0c40e969f23 so we'll pass R-CMD-check. I've just implemented the examples from `sun-methods.Rd`

**Review by adokter — APPROVED** [review]

**Pull request merged**